test/images/image-util.sh

```diff
@@ -142,11 +142,16 @@ push() {
 bin() {
   for SRC in $@;
   do
+  if [[ "${ARCH}" == "arm" ]]; then
+    arch_prefix="GOARM=${GOARM:-7} GOARCH=arm"
```

Comment:
why are we hardcoding `GOARCH=arm` when we know `GOARCH=${ARCH}` would give the same result?

this is more brittle due to relying on bash string splitting below.

Reply:
see https://github.com/golang/go/wiki/GoArm, this is only needed when ARCH is `arm`, it will make the info of ARCH more explicitly.

Reply:
what I mean is, we are unnecessarily branching the `GOARCH=` part which then causes us to rely on shell splitting below which is known to be a source of bugs. We don't need to do the GOARCH part here even if we want to use a branch to set GOARM.

_IF_ we did, we should use an array instead of a space separated string.

Reply:
yes, I see, this is not necessary.

Reply:
we should also consider removing the GOARM=7 in test/images/Makefile

we can find other places with GOARM with https://cs.k8s.io/?q=GOARM&i=nope&files=&repos=kubernetes/kubernetes

it seems to be hardcoded in a few places. I'm not sure on the history behind that, but we can find out!

Reply:
sure, will do, thanks!